clusterDeployer.py

```diff
@@ -462,6 +485,10 @@ def need_external_network(self) -> bool:
     def deploy(self) -> None:
         self._validate()
 
+        if self._cc.kind == "microshift":
```

Comment:
We are skipping pre and post config entirely?

Reply:
Why do you think this is the case?

Reply:
I am not sure what preconfig/postconfigs steps we will need in order to make this work. If it's rhel+microshift the ISO gets microshift up and running. I didn't need any more steps which is why I skipped pre and post configs. Do you think we need these?

Reply:
Yes we will most likely want to install the dpu-operator post Microshift installation. 4.15 of Microshift has proper operator and OLM support.

Reply:
Yeah you're right. Will make changes to address this.

Reply:
Can you click on "resolve conversation" if you're done with addressing a change?